apps/nodejs/extension/src/ui/ImportAccount/ImportAccountModal.tsx

```diff
@@ -175,22 +175,22 @@ export default function ImportAccountModal({
   }, [open]);
 
   const isCreateAccountDisabled = useMemo(() => {
-    const selectedNetwork = networks.find((n) => n.id === watch("protocol"));
+    const selectedNetwork = networks.find((n) => n.protocol === watch("protocol") && n.isProtocolDefault);
```

Comment:
If you want to get the values inside a callback, use the function `getValues`. `watch` is a hook, so you should declare a const and then use that const in the callbacks.